src/ImageCarouselReact/widget/components/ImageCarousel.tsx

```diff
@@ -143,7 +135,7 @@ export class ImageCarousel extends React.Component<ImageCarouselProps, {}> {
             );
         } else {
             // returned when there is no data/images
-            const classes = this.props.widgetId + " image-carousel-react image-carousel-nodata";
+            const classes = this.props.widgetId + " image-carousel-react image-carousel-no-data";
```

Comment:
Declaration of classes can moved above if statement, it is used in all 3 cases. Then add extra classes where needed.